internal/source/kubernetes/router.go

```diff
@@ -194,12 +194,12 @@ func (r *Router) mergeEventRoutes(resource string, cfg *config.Config) map[confi
 
 	// add routes related to recommendations
 	resForRecomms := recommendation.ResourceEventsForConfig(cfg.Recommendations)
-	r.setEventRouteForRecommendationsIfShould(&out, resForRecomms, resource)
+	r.setEventRouteForRecommendationsIfShould(&out, resForRecomms, resource, cfg)
 
 	return out
 }
 
-func (r *Router) setEventRouteForRecommendationsIfShould(routeMap *map[config.EventType][]route, resForRecomms map[string]config.EventType, resourceType string) {
+func (r *Router) setEventRouteForRecommendationsIfShould(routeMap *map[config.EventType][]route, resForRecomms map[string]config.EventType, resourceType string, cfg *config.Config) {
```

Comment:
1. Can we cover this change with integration tests? We already test the recommendations, but now let's ensure that we get only the events for a given namespace. So it could be just an extension of https://github.com/kubeshop/botkube/blob/main/test/e2e/bots_test.go#L978 - create same resource but in different ns and expect there's no recommendation notification about it.
2. Please also update docs to highlight that the recommendations now work like this. I think just a single sentence will do the trick: https://docs.botkube.io/configuration/source/kubernetes#recommendations